homeassistant/components/isy994/button.py

```diff
@@ -54,3 +57,27 @@ def __init__(self, node: Node | ISY, base_unique_id: str) -> None:
     async def async_press(self) -> None:
         """Press the button."""
         self.hass.async_create_task(self._node.query())
+
+
+class ISYNodeBeepButtonEntity(ButtonEntity):
+    """Representation of a device beep button entity."""
+
+    _attr_should_poll = False
+    _attr_entity_category = EntityCategory.DIAGNOSTIC
+    _attr_has_entity_name = True
+    _attr_entity_registry_enabled_default = False
```

Comment:
I thought about this and we enable the `Identify` buttons in HomeKit by default with them set to DIAGNOSTICS so I think it would be more consistent to enable them by default since they serve a similar purpose.

```suggestion
```

Reply:
Good with that.